src/Breadcrumb.js

```diff
@@ -1,27 +1,43 @@
 import classNames from 'classnames';
 import React from 'react';
+import PropTypes from 'prop-types';
 
 import BreadcrumbItem from './BreadcrumbItem';
 import { bsClass, getClassSet, splitBsProps } from './utils/bootstrapUtils';
 
+const propTypes = {
+  /**
+   * ARIA label for the nav, if no label is provided
+   * it will default to "breadcrumbs".
+   */
+  label: PropTypes.string
+};
+
+const defaultProps = {
+  label: 'breadcrumbs'
+};
+
 class Breadcrumb extends React.Component {
   render() {
-    const { className, ...props } = this.props;
+    const { className, label, ...props } = this.props;
     const [bsProps, elementProps] = splitBsProps(props);
 
     const classes = getClassSet(bsProps);
 
     return (
-      <ol
-        {...elementProps}
-        role="navigation"
-        aria-label="breadcrumbs"
-        className={classNames(className, classes)}
-      />
+      <nav aria-label={label}>
+        <ol
+          role="navigation"
+          {...elementProps}
```

Comment:
@taion do we have an opinion on how to spread through props? Always the root element? Maybe we should also add a `listProps` prop as well?

Reply:
we never did sort this out \:

shouldn't the styling props go to the root element, at least, though?

Reply:
i think so yeah.

Reply:
Just let me know how you want me to proceed. :D

Reply:
lets spread `elementProps` into the `nav` and add another prop `listProps` that is spread into the list element 👍 and in general going forward we'll try and be consistent on that pattern